src/EventListener/LoadDataContainerListener.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * Copyright (c) 2020 Heimrich & Hannot GmbH
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace HeimrichHannot\CategoriesBundle\EventListener;
+
+use HeimrichHannot\CategoriesBundle\Backend\Category;
+
+/**
+ * @Hook("loadDataContainer")
+ */
+class LoadDataContainerListener
+{
+    public function __invoke(string $table)
+    {
+        switch ($table) {
```

Comment:
Stelle hier sicher, dass er nicht immer wieder in den Callback geht (statische $run-Variable, die auf true gesetzt wird, wenn er anfängt ist).

Reply:
Ist doch ein Hook. Kann das passieren? Habe loadDataContainer-Hooks immer so gebaut. Theoretisch kann er in einem Ajax-Call nochmal reinlaufen, aber das wäre gewollt.

Reply:
Ja, das kann passieren. Hatte ich zumindest schon mal. Lass dir mal mit dbf eine Liste der $table-Werte geben. Da hatte ich schon teilweise viele Dopplungen drin.

Reply:
Ok, das ist komisch, darf eigentlich nicht passieren. Also laut dbf kommen keine Dopplungen und es wäre bei dieser Methode auch nicht schlimm, da hier ja nur die DCA angepasst wird.